test/e2e/atlas/cleanup_test.go

```diff
@@ -38,19 +39,27 @@ func TestCleanup(t *testing.T) {
 		"-o=json")
 	cmd.Env = os.Environ()
 	resp, err := cmd.CombinedOutput()
-	req.NoError(err, resp)
+	req.NoError(err, string(resp))
 
 	var projects mongodbatlas.Projects
 	err = json.Unmarshal(resp, &projects)
 	req.NoError(err)
 
 	for _, project := range projects.Results {
-		if project.ID == os.Getenv("MCLI_PROJECT_ID") {
-			fmt.Println("skipping project", project.ID)
-			continue
-		}
-		t.Run("Project "+project.ID, func(t *testing.T) {
-			deleteProjectWithRetry(t, project.ID)
+		projectID := project.ID
+		t.Run("deleting project "+project.ID, func(t *testing.T) {
+			t.Parallel()
```

Comment:
delete projects in parallel specially with all the wait for failing ones and now also deleting clusters and network stuff